sdk/python/kfp/dsl/_ops_group.py

```diff
@@ -93,9 +93,11 @@ def __enter__(self):
   def __exit__(self, *args):
     _pipeline.Pipeline.get_default_pipeline().pop_ops_group()
 
-  def after(self, dependency):
-    """Specify explicit dependency on another op."""
+  def after(self, dependency, *ops):
```

Comment:
For my education, why not merge `dependency` with `ops`?

Reply:
As described in [the issue](../issues/4787#issue-746812656), it's for it to remain backwards-compatible with by-keyword call (`group.after(dependency=op)`), considering that it wasn't a positional-only argument. I don't think many users used it, but maybe some did.

It does break no-argument call, which means unpacking a list might throw an error (`group.after(*[])`), but I assume it's less of a problem, considering it's wasn't supported before either.

We can remove it if you think it's not really used by any users.

Reply:
Thank you for taking the backwards compatibility seriously. This is a great approach.

In this case I think it's pretty safe to make the change. The `OpsGroup` is not considered to be public and the `OpsGroup.after` is likely not used widely. In cases where it's used, it's likely to be used without positional argument.

Reply:
@Ark-kun Good to know! If that is the case, then using args directly is probably a better idea, as it's simpler and works safely for any list.